src/collections/_documentation/workflow/discover2/query-builder.md

```diff
@@ -0,0 +1,333 @@
+3. Optionally, add an aggregate function
+4. Click "Create column" button
+5. Your new column will be added to the right-most side of the table. You may need to scroll towards the right to be able to see it.
+
+### Edit Columns
+
+1. Click on "Edit Columns" on the top right-hand corner of the table to enter the edit state of the table
+2. Hover your mouse cursor over the header of the desired column to edit
+3. Click on the pencil icon
+4. A modal will open for you to edit the column
+5. Click the "Update column" button
+6. Optionally, click on the "Save & Close" button on the top right-hand corner of the table to exit the edit state of the table
+
+### Delete Columns
+
+1. Click on "Edit Columns" on the top right-hand corner of the table to enter the edit state of the table
+2. Hover your mouse cursor over the header of the desired column to delete
+3. Click on the garbage bin icon
+4. The table may refresh
+5. Optionally click on the "Save & Close" button on the top right-hand corner of the table to exit the edit state of the table
+
+### Resize Columns
+
+You can resize columns by hovering over the column boundary, and clicking & dragging the boundary to resize the column to the desired width.
+
+### Sorting Columns
+
+1. Hover your mouse cursor over the header of the desired column to sort
+2. Click on the header
+3. The table may refresh
+
+A down arrow indicates sorting the column in descending order. An up arrow indicates sorting the column in ascending order.
+
+### Download CSV
+
+Note, only the current values visible on the table are exported, not the paginated results. To export the table as CSV, click on the "Download CSV" button.
+
+### Expand Row Aggregates
+
+Many queries will result in observing aggregated data. Some typical results might look like the count of errors rolled up by project, URL, or release.
+
+Seeing these aggregations will indicate where to focus your attention. You can click on any cell in the `count(id)` column and expand the results from that specific row. In doing so, you can view the individual events within that aggregation and see the tag distribution, event volume, or continue to drill down and investigate further.
+
+### Facet Maps
+
+Each query result is displayed in tandem with a facet map that represents a distribution of tags that corresponds with the query result.
+
+[{% asset discover/discover-facet-map.png alt="Facet map for transaction and handled tags. Map looks like a bar with gradient colors." %}]({% asset discover/discover-facet-map.png @path %})
+
+The top 10 tag keys and their values are shown by occurrence and dynamically update when you change any of your conditions. For example, if you choose a different project or date range, the facet map will automatically update based on the new results. 
+
+You can also hover over each value to see the exact distribution for a given tag and click on them to add to your filter conditions and see a new query result.  
```

Comment:
Have `filter conditions` link to the sections about filtering seems like a good idea.

Reply:
Links are cheap!

Reply:
Done!